### ISSUE TITLE
"Ophthalmology" as MedicalSpecialty

### ISSUE THREAD

**Issue by MikiDi** [schemaorg/schemaorg#1598]
Would you consider adding `Ophthalmology` as a [MedicalSpecialty](http://health-lifesci.3-2e.schemaorgae.appspot.com/MedicalSpecialty) in the health-lifesci vocabulary? This seems like a quite common medical speciality that is missing.
This was already suggested once [here](https://github.com/schemaorg/schemaorg/issues/492#issuecomment-213035147). It has also been [proposed](https://github.com/schemaorg/schemaorg/issues/492#issuecomment-213429305) to change "optometric" to "ophthalmology".

ps: out of interest ... It looks like quite some good suggestions were made in #492 ([here](https://github.com/schemaorg/schemaorg/issues/492#issuecomment-213429305)) regarding completing the list of medical specialities and making the naming consistent (some related issues are also being discussed in #1062 ). Are these suggestions still considered/discussed for a future release?

cheers,
M.

tracking: #1116

**Comment by twamarc** [comment]
Thanks raising this issue @MikiDi !
We had a huge discussions on ``` MedicalSpeciality``` types and the decision was made to add speciality types based on use-case needs (as they show up). There is one comprehensive document drafted and discussed here:
https://docs.google.com/document/d/151dHBdBcVJ6cW9uqT3ARrmLDFO9gPPSnh87ed4KC844/edit#heading=h.uqaolvli18fz

I think this guiding document is mature enough for implimentation in our vocab **BUT** we need to pick only types  **based on a real demand of use cases**- we can't add so huge amount of types at once without a **solid hope of future use/implimentaion** (even if we suppose they are usefull).
Do you have some use-cases where a SEO or a webmaster or an API (or a domain) needs this type in particular? I would be happy to see.
I am confident that with gradual increase (time-to-time) and availability of use-cases/implimentations, all the types listed in the document will end up in our vocab.
Meanwhile feel free also to read and comment the document.

My views!
Marc

**Comment by MikiDi** [comment]
Thanks for your fast reply!
My (admittedly limited) use case would be to be able to add the speciality as schema markup when designing a website for a specialist doctor's practice. This could probably be categorised as an (indirect) SEO technique although I think it could become useful in the future for more than just that.   

I understand the concern of polluting the namespace with terms that have no real-world use-case. Nonetheless I think that (especially for the most common specialities) **adoption will follow automatically when you provide a complete & well-documented vocabulary**. 

Maybe a good **starting point for deciding which specialities to add** could be to imagine the use case of an average hospital that wants to add schema markup to the list of its doctors and their specialities?

**Comment by twamarc** [comment]
I would not start by **imagine** a hypothetical use-case myself, even if it is viable or it might be important in future.
The philosophy is to start by an existing real use-case and see the needs there and then advocate to add them into the vocab.
few examples:  
1° Some collaborations will -in near future, require additional types and models. Such collaborations include -but are not limited to: [BioSchemas](http://bioschemas.org/), the more advanced [US Health Insurance vocab](http://pending.schema.org/HealthInsurancePlan), and others.
2° When we were collaborating with [ SALUS project ](http://cordis.europa.eu/project/rcn/100716_en.html) a couple of types and models were suggested to be added to the vocabulary so the team could formalize their EHRs data.
3° Cleveland clinic started marking up somes of their pages... this is real use-case in case they have a missing type or a wrong model for instance.

Those are few use-cases potentially behind each of the types requests. This means that each request should be based on a real use-case.

my 2 cents
Marc

**Comment by LeezaRodriguez** [comment]
Oye-the mention of 'good suggestions made in #492 ' has raised my cortisol level. Yes, it's hard to believe , but I still get worked up about this. Please allow me to repeat some old concerns:

### Collaborating with others: US Health Insurance vocab, Clinical Trial database, etc
As long as there is not a [distinction between the physician person and the physician organization](https://github.com/schemaorg/schemaorg/issues/280#issuecomment-253915688), good luck with that.  Physician organization and physician individuals have different unique NPI numbers which are issued by CMS here in the USA.  Insurance carriers *revolve* around NPI numbers. Once the vocab  introduces 'participating providers' this is going to get hairy.  The Physician person works for a Physician org, which each have distinct NPI numbers. Also , a physician person can work for multiple physician orgs.  Yikes.  Likewise, the world of Clinical Trials revolves around the PI, or Primary Investigator. This is most often times a physician-person and not an org.  I just don't see how any of this scales without disambiguating the person from the org.

### Real Use Cases, Ex: Cleveland Clinic
About 18  months ago I found [3 examples of major teaching institutions](https://github.com/schemaorg/schemaorg/issues/807#issuecomment-144743762) (including the Cleveland Clinic) who do not appear to be using `Physician` correctly . AFAICT, these websites devote a unique webpage to each individual doctor. In other words-- the common use case for all of these large organizations is their website *doctor directory*.  BUT , each doctor is *not* an organization as `Physician` currently defines, but rather each person in the directory is a physician-person with his own personal [NPI number](https://github.com/schemaorg/schemaorg/issues/806#issuecomment-144555820) (NOT his organization NPI number), and with a `MedicalSpecialty` that he is Board Certified in.    Thus, although `Physician` is being used as the data type, these websites should be using a data type for the individual doctor person. 

### `MedicalSpecialty`
Truth or consequence time. This data lineage starts with the physician person, not the organization. A physician person completes a residency training in a Medical Specialty and attains Board Certification to practice that `MedicalSpecialty` . The org inherits the Medical Specialty by virtue of the doctors employed in the unit. Whenever a new physician-person data type is created, there should be a property with an expected type = `MedicalSpecialty` and some ET mediation with the concept of Board Certification, as well as type of Medicine practiced (western conventional, osteopathic, etc).  

The current `MedicalSpecialty` enumerated list includes adjectives and nouns,  instead of all nouns. And even overlooking this fact, the current `MedicalSpecialty` enumerated member list  includes only 20 of the [37 formal ABMS  top level MedicalSpecialty](http://www.abms.org/member-boards/specialty-subspecialty-certificates/) classifications. That is just a crying shame for an industry that represents 17% of US GDP. The truth is that most teaching and hospital departments in the USA are named according to ABMS Medical Specialty. Orders for lab tests, diagnostics, supplies, and treatment care flow down from the physician person. Doctors certified in a MedicalSpecialty train to treat specific diseases or to perform certain surgeries. `MedicalSpecialty` is the top of the food chain.  So, why can't we get this right?

Is it any wonder why the doctor directory use cases do not make more use of `MedicalSpecialty`? To start with, the data type as defined by schema.org  is currently only applied to orgs, *not individuals* . But even if you misuse the data type, how could a webmaster with a list of hundreds of individual doctor pages commit to an enumerated list if only half of the MedicalSpecialties are enumerated ??  And what about the other 84 ABMS Medical Subspecialties? Why not just combine the list of ABMS Medical Specialty and Medical Subspecialty and call it a day? I dare you to find a topic that does not fit among these Medical Specialties!  Stem cell science you say?  AHHH--that is simply a new delivery mechanism for EACH AND EVERY Medical Specialty!   It is NOT a new MedicalSpecialty. 

In a nutshell, the current enumerated member list is deficient. We should make this comprehensive , ASAP.


### [The working draft proposal](https://docs.google.com/document/d/151dHBdBcVJ6cW9uqT3ARrmLDFO9gPPSnh87ed4KC844/edit#heading=h.uqaolvli18fz)- why so many data types ?
Once upon a time , @twamarc thought of a [simple way to fix this physician-person vs physician-org issue](https://github.com/schemaorg/schemaorg/issues/492#issuecomment-213704477) in the Medical extension. To elaborate:
1) create a new Physician-person data type with associated properties for board certification, medical specialty, NPI number, license/jurisdiction, Residency, etc. 
2) expand the `MedicalSpecialty` member list to either 37 ABMS members , or 37 + 84 members (all ABMS Medical Specialties and Subspecialties)

To make a long story short, new tangential discussions emerged about creating a `Profession` data type,  and the original pull request for Marc's new data type in the medical extension was declined.  Fast forward a year later ....
We tried to come up with a solution which would address our medical extension needs (physician person vs org) , as well as introduce a data type that would allow for a professional resume to be marked up. It also allowed some critical functionality such as to allow `Reviews` on a Profession and not just an org.  Generally speaking, it is the doctor-person who has patient reviews at Healthcare portals, not the org. 

But we might be in over our head. And the ability to dedicate more time to this is an issue for all of us.

In the end, I don't care which way we go. We just need to go.

### Old rants-in case you missed it
1- [In The Wild](https://github.com/schemaorg/schemaorg/issues/1062#issuecomment-277072935)
2- The [need to disambiguate](https://github.com/schemaorg/schemaorg/issues/280#issuecomment-253915688) the Physician office from the Physician person
3- [`MedicalSpecialty`](https://docs.google.com/spreadsheets/d/1y7KIP0icoqowZ1uX4H33rVJKfj5WAjIeqkEpnuN6A9U/edit#gid=0) spreadsheet

p.s. Sorry, I can't help myself.

**Comment by tjmb** [comment]
I'm posting a response here because I have what I would consider to be a "real demand of use case" for Ophthalmologist/Ophthalmology as a specific type of MedicalSpecialty. I host and maintain a set of websites for an ophthalmology office. The doctors there recently discovered that their previous web developer had use the term "Optometrist" in the website and have requested that I change all of those occurrences to "Ophthalmologist" because "Optometrist" is not the correct term for the services they provide.

Some of these uses of Optometrist are in schema.org data and in the process of correcting the type from "Optician" to "Ophthalmologist" I discovered that officially there is no such data type.

If there are already two data types of Optician and Optometric, neither of which are accurate for this use case, why can't one be added for Ophthalmology?

> **Ophthalmologists** differ from **optometrists** and **opticians** in their levels of training and in what they can diagnose and treat. As a medical doctor who has completed college and at least eight years of additional medical training, an ophthalmologist is licensed to practice medicine and surgery.
Source: [Difference between an Ophthalmologist, Optometrist and Optician](https://aapos.org/terms/conditions/132) - American Association for Pediatric Ophthalmology and Strabismus

**Comment by thadguidry** [comment]
@tjmb use the Schema.org type of Physician or another appropriate type and add additionalType: https://www.wikidata.org/wiki/Q36345695
or if your need is for the business providing a Service, then add additionalType: https://www.wikidata.org/wiki/Q161437

Schema.org has many types, but cannot cover every type in the world (the Long Tail domains like medicine)...so use a higher type already provided and then add additionalType and perhaps sameAs to give more information about your structured data type.  Search Engines and Apps will understand.

**Comment by tjmb** [comment]
Thank you @thadguidry, that advice is much appreciated! I'm just becoming familiar with schema.org and didn't realize that a custom additionalType could be added. Great to know!

**Comment by westurner** [comment]
Which file does Ophthalmology need to be added to?

Here's this from https://github.com/schemaorg/schemaorg/blob/main/data/ext/health-lifesci/med-health-core.ttl :

```turtle
:Cardiovascular a :MedicalSpecialty ;
    rdfs:label "Cardiovascular" ;
    :isPartOf <http://health-lifesci.schema.org> ;
    rdfs:comment "A specific branch of medical science that pertains to diagnosis and treatment of disorders of heart and vasculature." .

:CardiovascularExam a :PhysicalExam ;
    rdfs:label "CardiovascularExam" ;
    :isPartOf <http://health-lifesci.schema.org> ;
    rdfs:comment "Cardiovascular system assessment withclinical examination." .

```

**Comment by westurner** [comment]
Should the pull request contain the phrase "closes #1598"? That would result in this issue being automatically closed when or if the proposed PR is merged.
https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

**Comment by b-ro702** [comment]
TL/DR Is there any movement on this? It's a well documented well sought after service and surgical specialization.

I am working with a large medical health organization that would like to appropriately markup their clinics that offer ophthalmology as a service so I can affirm their is demand for support.

I can't help but wonder how we got Optician (a technical practitioner who designs, fits and dispenses corrective lenses for the correction of a person's vision) but ophthalmologist (a doctor that diagnoses and treats all eye diseases, performs eye surgery and prescribes and fits eyeglasses and contact lenses to correct vision problems) is too specific to be worth including.

Pardon my incredulousness, but that logic seems lazy considering defining Public Toilet (which has 0 unique properties) was worth prioritizing as a unique type and not an enumeration of civic structure and 3 years later we can't be bothered to define a well known and often used medical specialty as an enumeration. 

I'd further add that arguments above saying that there wouldn't be enough adaptation to support creation create a nice vicious circle to defend the unwillingness to expand this vocabulary. If you don't build the enumeration or type out, then it becomes challenging for website owners to mark up the site meaning they won't mark the site/related pages up. How do you determine what would and would not be highly adapted? As of right now, it seems arbitrary.

I get that schema.org can't list EVERY last possible type, but after three years one would think this a reasonable request that could impact a large swath of medical websites/practitioner websites/pages.

**Comment by westurner** [comment]
Which file needs to be edited to make this happen?

**Comment by LeezaRodriguez** [comment]
There was a post on Dec 3rd expressing frustration with the enumerated list of [MedicalSpecialty](https://schema.org/MedicalSpecialty).  I can no longer see the post.  I too share this frustration. I have posted about this for many years, right here on Github, only to see the threads closed by humans or bots.  A few of us had spent quite a bit of time studying this issue and working on a proposal that would tip toe around the issues that exist. To my knowledge, nothing has been addressed over the past few years. 

The enumerated MedicalSpecialty list is currently unable to properly serve the medical community.  It is missing many of the top level **Medical Specialites** as defined by the ABMS (American Board of Medical Specialties), as well as scores of **Medical Subspecialties**. The enumerated list is currently a mash up of nouns, adjectives, and occupations. It is NOT a proper Medical Specialty list. 

 For many years, hospital webmasters have either they had to go to external enumerations, or just call it quits, as you can't only markup half of your staff.  In addition this enumerated list for MedicalSpecialty is  available only for type: /Physician, the org place of business.  It should be attached to the Physician-person. 

In the US, the vast majority of medical doctors get their  Medical Speciality certification from one of 24 Boards regulated by the American Board of Medical Specialties (ABMS). Using [ABMS website](https://www.abms.org/member-boards/specialty-subspecialty-certificates/), we can easily edit the current enumerated list of Medical Specialties to include:
36 Medical Specialties
84 Medical Subspecialties

The simplest mechanism would be to either a) use only 36 Medical Specialties b) collapse these Medical Specialties and Medical Subspecialties together.  This is not the ideal solution, but better than what we have.  In a perfect schema.org world  there would be a MedicalSubspecialty type or a property. However, we MUST also tie these Medical Specialties to the Physician-person, which does not currently exist.  The [type: Physician](https://schema.org/Physician) is currently a place or org, and NOT a person. Therefore,  the [type:MedicalSpeciality](https://schema.org/MedicalSpecialty) is only used in conjunction with Physican, the org or place. It is not applicable to a person. This is a problem.

IMPORTANT CONCEPTS:
- **Medical Specialty provenance:** A Medical Speciality originates lineage and provenance from the physician person, NOT the organization. It is the Physician- person who gets Board Certified. The org does NOT get Board Certified.  The org only inherits the Medical Speciality from the individual medical doctor persons who work in the org.  We need to tie the new and improved MedicalSpecialty enumerated list to a MedicalDoctor person. 
-  **What is a Medical Specialty**?:A Medical Specialty is not self proclaimed expertise. It is not a loosey goose thing.  A Medical Specialty has a Medical Board who provides oversight to the Board Certification of physicians in a particular Medical Specialty. It defines the training, exam requirements, and continuing education requirements.   And in the ABMS, these 24 individual Medical Boards  (overseeing 36 Medical Specialties) have an oversight board, the ABMS. This is highly controlled and regulated!  
-  **NPI numbers**: In the US, the disambiguating identifier for any medical entity is the **NPI number.**  The Physician-person and the Physician-org **EACH have separate unique NPI numbers.**  Each of these NPI numbers (individual physician and org) are included on claim forms to Medicare, Medicaid, as well as most insurance carriers.   NPI numbers also link to **Medical Board sanctions against a physician, or a license revocation**.  Malpractice data bases also use NPI numbers. If you don't have an NPI #  in the US, you don't exist .

ISSUES and IMPLICATIONS FOR THE CONSUMER

1- Parsing out MedicalSpecialty
Snippets and answer boxes should be able to parse the MedicalSpecialty and Medical Subspecialty of any Physician-person.  It is a crime that we can get an elaborate parsing of a recipe, but the credentials of a doctor who will be treating your very sick child or parent can not be parsed.  If this were properly done, it could parse like this:

**Anthony Fauci, MD**
**Medical Specialty**: Internal Medicine
**Medical Subspecialty**: Infectious Disease

2- CMS transparency- NPI numbers are the disambiguating link!
As CMS pushes forward with transparency and API interoperability, we need a way to disambiguate the Physician-person and their respective Physician-orgs.  Again, in the US this is tied to NPI numbers. While NPI can be added now as a property for the Physician-org type, **it must be available for a Physician-person.**  When I last researched this a few years ago, the medical web was a mess of webmasters erroneously using NPI INDIVIDUAL person # on the /Physician org type. A  hot mess.

3- Clinical Trails
 Don't we want to be able to easily markup Primary Investigators in Clinical Trials? These are individual Physician-persons who lead the study.  And don't we want to know that doctor's Medical Speciality?  If it is a trial for COVID Pulmonary treatments, don't we think it important that the PI's Medical Speciality is parsable? In this example, we might expect it to be: 
**Medical Specialty**:  Internal Medicine
**Medical Subspecialty**: Pulmonary 

Medicine is already fragmented enough. The markup to disambiguate physicians and detail their Medical Specialty certification is essential to helping the consumer sort this out . This is doable. 

thanks,
Leeza

**Comment by westurner** [comment]
Thanks for explaining.

Is `med-health-core.ttl` the file that we need to add turtle to?
https://github.com/schemaorg/schemaorg/blob/main/data/ext/health-lifesci/med-health-core.ttl

And an adjacent MedicalSpecialty-examples.txt?

Who would need to approve a PR with these changes? @danbri @Dataliberate

**Comment by westurner** [comment]
Are there already (usable) descriptions of each {ABMS,} MedicalSpecialty?

**Comment by LeezaRodriguez** [comment]
> Are there already (usable) descriptions of each {ABMS,} MedicalSpecialty?

Thank you for taking the time to read, and acknowledging!

These definitions could be extracted from the first paragraphs of the Medical Specialty and Subspecialty links at the [ABMS website](https://www.abms.org/member-boards/specialty-subspecialty-certificates/).  Once this gets decided ( 36 or 120 enumerated ?) , I would be willing to participate in populating this description info. But I don't want to waste my time -- 5+ years ago I entered **ALL of these definitions** into a Freebase Medical Specialty staging area, but that work went up in smoke with the Wikidata merger.  I had mapped the various pathways to all the Medical Subspecialties from each Medical Specialty.  In many instances, there is more than one pathway to any Medical Subspecialty.

I am pretty busy managing a surgery center during a pandemic, but this is too important if the powers-that-be agree to fix it. 

Lastly, @jvandriel and @thadguidry had given a lot of thought to various work-arounds (use of new type: /Profession) for the issue with a Physician-person type not existing.  Likewise,  another route considered in the past was a new type: /MedicalDoctor , which could assign a property with ET: MedicalSpecialty, and a new NPI property. The NPI property should also be added to the Physician-org. 

Just thinking out loud. I have not been in the loop for years, so I don't remember much of the nuances.  Thanks again.

**Comment by westurner** [comment]
Could we generate some turtle for med-health-core.ttl by scraping https://www.abms.org/member-boards/specialty-subspecialty-certificates/ and either creating rdflib objects or just string template?

Good luck with the zombies,
Out of the loop as well.

**Comment by twamarc** [comment]
@westurner: once the PR is ready post it here for review... please add potential community uptake to convince the necessity of this 🔢

**Comment by LeezaRodriguez** [comment]
If you are willing to scrape, and can put the strings in a spreadsheet, I can edit.

BTW, @twamarc , **hi!** As I recall, you had once written a PR for adding a type to represent a physician person. And then something something happened.  Does my memory serve me right?  And with the zombies, lol, they were both tweeting about this a few days ago.  They are just pretending to be zombies.

**Comment by thadguidry** [comment]
@LeezaRodriguez @westurner I know some of that is tongue-in-check and meant to be humorous, but let's be careful on word choice and also realize that there are numerous contributors and that everyone needs to adhere to the W3C community policy (respectful) even here in GitHub since this is where communication happens as well and we decided it's subject to the same policy. https://schema.org/docs/about.html

**Comment by twamarc** [comment]
@LeezaRodriguez : my PR didn't collect enough community enthusiasm for re-use in real cases unfortunally and was abandoned.

**Comment by westurner** [comment]
I have confidence that you can get this done.

**Comment by danbri** [comment]
Hi folks. Thank you for all your energy and enthusiasm for improving this markup. Let me try to give an overview from schema.org's perspective on where we are, and why I didn't pull in further changes to this section of the project in recent years.

Basically we added a *huge* number of terms back in 2013, and despite @twamarc and others' efforts to improve their quality, integrity and definitions, much of the medical and healthcare vocabulary remains in very large part underutilized (or even, effectively, un-used), with regard to applications that actually do something tangible with the data.

On the Google side so far, we do not have much structured data functionality in this area - i.e. nothing along the lines of the features in https://developers.google.com/search/docs/guides/search-gallery. This does not mean that such markup is completely unused - it could probably be called "weakly used" in contrast with these higher profile Search features.

(This experience btw was one of the reasons schema.org now has the "pending" area, to give proposals freedom to grow before being added to the core of schema.org.)

Earlier this year I polled the schemed community group, looking for evidence of other *[consuming](https://lists.w3.org/Archives/Public/public-schemed/2020Jan/0000.html)* applications for this data, but nothing came up. Without significant usage it is difficult to progress schemas without some grounding in real world needs, so I have resisted tinkering further with the vocabulary until there are signs of substantive use.

# 2021 planning

I think that time may have come.  Last week we made a [guest post](http://blog.schema.org/2020/12/modernizing-us-health-provider-sites-to.html) on the schema.org blog from Aneesh Chopra. It seems that in a US setting at least there is significant demand for something in this area (and potentially for alignment with standards like FHIR). We also need to remember that the schema.org vocabulary serves an international role, so we should take care to be clear if we define US-specific structures. Some recent structural changes on schema.org have made this easier. For example see https://schema.org/NonprofitType where an Enumeration has 3 country-specific subtypes. For short lists that could be an option. For longer lists like https://www.abms.org/member-boards/specialty-subspecialty-certificates/ we could perhaps have a dedicated property that takes textual or URL values, to avoid adding dozens more terms into schema.org.

I would like to hear more from the US group that Aneesh is convening before jumping to a specific design conclusion here, but I think it's a good time also to say "we hear you!" with respect to the frustrations that people have had with the vocabulary in its current state. Ultimately schema.org itself can only provide a dictionary of terms that make structured data descriptions of things _possible_, richer and detailed best practice guidelines can only really be created when someone is using that data for concrete applications. This transition has happened before with schema.org terms like JobPosting and Dataset, and I think there are good signs we may be approaching such a point for at least some of schema.org's medical / healthcare-related terminology.

**Comment by chrisspradling1980** [comment]
*It's nice seeing this shift in technology. More expansion for money in the
long run~-------------------|*

Greater LEG, DIG!!!!
-Thank you respectfully-
Christopher Michael Spradling

**Comment by LeezaRodriguez** [comment]
@danbri --Thank you very much for updating us! Glad to hear about the renewed interest and I have signed up to join the effort. I would very much like to help.

It is very exciting to see the mention of [NPI numbers](https://npiregistry.cms.hhs.gov) on Aneesh's blog post.  This certainly sets the stage for the need to identify the NPI#  of both the individual Physician (or healthcare provider) AND the NPI # of the Orgs that they belong to.  

Furthermore, if one enters the individual NPI number for a individual physician, the search result returns the **Primary Taxonomy** = Medical Specialty.  If I further click thru this Taxonomy link for an individual physician, we can see that the [Taxonomy description](https://npiprofile.com/taxonomy/code/208200000X) is sourced from the **ABMS**.   You can probably glean a lot of other info about the CMS classification system from this one screen . :-) 

Onward to 2021!  

cheers,
Leeza

**Comment by westurner** [comment]
Should there be a datetime attribute on the relation between a Professional
and an Organization (each with a distinct NPI https://schema.org/identifier
FWIU)?

**Comment by thadguidry** [comment]
@westurner [Roles](http://blog.schema.org/2014/06/introducing-role.html) already help there.  Why don't you mockup a JSON-LD snippet here for us using Role for all to see.  You can do it! :-) I'm lazy today.

**Comment by westurner** [comment]
Yeah those were already called "qualified relations."
https://patterns.dataincubator.org/book/qualified-relation.html

RDFstar (and SPARQLstar) have special syntax for reifying relations.

Without a date on the Credential, data aggregators could create a false sense of security if it's possible to lose these specialty certificates. Are these certificates revoked in conjunction with a medical license in the US?

W3C Verified Claims is the spec for making Credential claims.

**Comment by westurner** [comment]
https://w3c.github.io/vc-use-cases/#professional-credentials
> C.1 Find a doctor
> Jason is looking for a new primary care physician. His health provider includes information on their web site about the physicians they have on staff, including verifiable credentials about their education, board certification, and continuing education. Jason can verify these credentials and be confident that his new physician satisfies his requirements.

C.2 Busy doctor
> Barney was a board-certified physician, but he ran out of time to complete his continuing education requirements and his certification lapsed. Since the board can revoke his certification, credential verifiers will automatically be aware that he can no longer issue prescriptions or perform medical procedures.

**Comment by westurner** [comment]
Ophthalmology MedicalSpecialty is still not merged? Would still be great to
have Opthalmology merged into the next release of schema.org RDFS schema

***

Also, W3C Verifiable Credentials and DIDs specifically specify professional
licenses in their Use Cases. Blockcerts already implement W3C Verifiable
Credentials (and registerable DIDs) and so could be used for medical
professional licensure.

https://www.w3.org/TR/vc-use-cases/#professional-credentials

https://www.w3.org/TR/did-use-cases/#vcEcosystem

In terms of JSONLD RDF, W3C VC specifies the signature object attr name to
remove from a JSONLD document before normalizing it by sorting before
cryptographically signing it, and the RDFS schema of said signature object;
with somewhat future-proof URIs for the signature algorithm.

Schema.org properties like medicalSpecialty could be included in JSONLD
cryptographically signed with a DID listed in a Verifiable Credential, IIUC.

The license validity date interval specified in a granted Verifiable
Credential could be verified by block height date lookup, or signing key
validity interval; If the JSONLD cert signature is valid, is the license
expiry date listed therein valid, too?

**Comment by westurner** [comment]
This is still not done

**Comment by jlndr** [comment]
Closed with https://github.com/schemaorg/schemaorg/pull/4808
